packages/list/README.md

```diff
@@ -108,29 +109,37 @@ class MyApp extends Component {
     );
   }
 }
+
 ```
 
 ### List groups and list dividers
 
 Multiple related lists can be grouped together using the `ListGroup` component. Optional subheaders can be added using `ListGroupSubheader`. `ListDivider`s can be used to separate content either within a list or between lists.
 
 ```js
-import React, {Component} from 'react';
-import List, {ListItem, ListItemText, ListGroup, ListGroupSubheader} from '@material/react-list';
+import React, { Component } from "react";
```

Comment:
import React, {Component} from 'react';

Reply:
I see, I didn't realize it changed. It was because I used [SublimeJsPrettier](https://github.com/jonlabelle/SublimeJsPrettier) a code formatter. Maybe you can suggest me another code formatter which follows google's best practices or code formatter configuration or also their guidelines. Besides in this particular example, what is the convention to put single quotes?

Reply:
Done

Reply:
Whenever its JS you should be using single quotes. I prefer single quotes in HTML, but some like to use double. In the typescript update, I'm adding the single quote eslint rule. :)